spec/gotest_spec.vim

```diff
@@ -149,4 +149,20 @@ describe "GoTest"
 
     Expect g:test#last_command == 'go test ./...'
   end
+
+  it "runs test with args"
+    view +5 normal_test.go
+    let g:test#go#gotest#args = "-args a=b"
```

Comment:
Given that the option is called `args` could we make it automatically add `-args` so you could just do the following?
 
```suggestion
    let g:test#go#gotest#args = "a=b"
```

Reply:
sure. will do

Reply:
done